src/shared/components/proteinChainPanel/PdbChainTable.tsx

```diff
@@ -0,0 +1,153 @@
+import PdbChainInfo from "../PdbChainInfo";
+import PdbHeaderCache from "../../cache/PdbHeaderCache";
+import {computed} from "mobx";
+import {PdbHeader} from "../../api/generated/PdbAnnotationAPI";
+import OrganismColumnFormatter from "./column/OrganismColumnFormatter";
+import LazyLoadedTableCell from "shared/lib/LazyLoadedTableCell";
+import {generatePdbInfoSummary} from "../../lib/PdbUtils";
+
+class PdbChainTableComponent extends LazyMobXTable<IPdbChain> {}
+
+export interface IPdbChainTableProps {
+    dataStore:PdbChainDataStore;
+    cache?:PdbHeaderCache;
+};
+
+@observer
+export default class PdbChainTable extends React.Component<IPdbChainTableProps, {}> {
+    @computed private get columns():Column<IPdbChain>[] {
+        const ret:Column<IPdbChain>[] = [];
+
+        ret.push({
+            name: "PDB Id",
+            render:(d:IPdbChain)=>{
+                return (
+                    <a href={`http://www.rcsb.org/pdb/explore/explore.do?structureId=${d.pdbId}`}
+                       target="_blank"
+                    >
+                        {d.pdbId}
+                    </a>
+                );
+            },
+            sortBy:(d:IPdbChain)=>{
+                return d.pdbId;
+            },
+            filter:(d:IPdbChain, filterString:string, filterStringUpper:string)=>{
+                return (d.pdbId.toUpperCase().indexOf(filterStringUpper) > -1);
+            }
+        });
+
+        ret.push({
+            name: "Chain",
+            render:(d:IPdbChain)=>{
+                return (
+                    <span>
+                        <span style={{float:"left", fontWeight:"bold", fontStyle:"italic"}}>
+                            {d.chain}
+                        </span>
+                        <a style={{float:"right", fontWeight:"bold", cursor:"pointer"}}
+                           onClick={()=>{ this.props.dataStore.selectUid(this.props.dataStore.getChainUid(d)); }}
+                        >
+                            3D
+                        </a>
+                    </span>
+                );
+            },
+            sortBy:(d:IPdbChain)=>{
+                return d.chain;
+            },
+            filter:(d:IPdbChain, filterString:string, filterStringUpper:string)=>{
+                return (d.chain.toUpperCase().indexOf(filterStringUpper) > -1);
+            }
+        });
+
+        ret.push({
+            name: "Uniprot Positions",
+            render:(d:IPdbChain)=>{
+                return (<span>{`${d.uniprotStart}-${d.uniprotEnd}`}</span>);
+            },
+            sortBy:(d:IPdbChain)=>{
+                return [d.uniprotStart, d.uniprotEnd];
+            },
+            filter:(d:IPdbChain, filterString:string)=>{
+                return (((d.uniprotStart + "").indexOf(filterString) > -1) ||
+                    ((d.uniprotEnd + "").indexOf(filterString) > -1));
+            }
+        });
+
+        ret.push({
+            name: "Organism",
```

Comment:
Would it be too difficult to move some of the functions defined here into `OrganismColumnFormatter`. Considering that they have dependencies to `this.props.cache`, it may not be straightforward to implement them there. If too much work, just ignore this comment :)